scripts/Muon/GUI/FrequencyDomainAnalysis/frequency_domain_analysis.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, parent=None, window_flags=None):
         self.help_widget = HelpWidget(self.context.window_title)
 
         central_widget = QtWidgets.QWidget()
+        central_widget.setMinimumWidth(590)
```

Comment:
Can you try putting this in the `setup_tabs()` function instead?
```
        self.tabs.setElideMode(QtCore.Qt.ElideNone)
        self.tabs.setUsesScrollButtons(False)
```
If it works, that would avoid any problems in the future if someone renamed a tab, for example.

Reply:
This works brilliantly and I agree - a far better approach as it will work with any future changes. I hadn't been happy with a fixed width but wasn't sure how else to resolve the problem. Many thanks.